pb/protos/profile.proto

```diff
@@ -56,4 +49,20 @@ message Profile {
         string large    = 4;
         string original = 5;
     }
+
+    message Stats {
+        uint32 followerCount  = 1;
```

Comment:
why are we using in some abstract types fields starting with capital letters (like on the `Profile`), and then we go java style `camelCase` on this one?

Note: I'm new to golang, just wondering, is it because this is a `message` vs the other ones being a `struct`.